src/main/flight/pid.h

```diff
@@ -184,3 +184,7 @@ void pidAcroTrainerInit(void);
 void pidSetAcroTrainerState(bool newState);
 void pidInitSetpointDerivativeLpf(uint16_t filterCutoff, uint8_t debugAxis, uint8_t filterType);
 void pidUpdateSetpointDerivativeLpf(uint16_t filterCutoff);
+void pidUpdateAGThrottleFilter(float throttle);
```

Comment:
`pidUpdateAntiGravityThrottleFilter()` would be more consistent.

Reply:
I originally had that but thought the function name was getting pretty long. ;)  I'll fix.